docs/DeveloperGuide.adoc

```diff
@@ -1119,6 +1119,48 @@ These instructions only provide a starting point for testers to work on; testers
 .. Prerequisites: Weme is in the memes tab and just executed a `find` command. An image file is on the hard disk.
 .. Similar to above, but after successful execution of `add`, the user is redirected to the `list` view.
 
+=== Editing a meme
+
+. Editing a meme while all memes are listed
+
+.. Prerequisites: Weme is in the memes tab.
+.. Test case: `edit 1 d/new description t/tag` +
+   Expected: The meme in the given index is edited. The view is updated to reflect this change. The status message states the edit was successful.
+.. Test case: `edit 1` +
+   Expected: No meme is edited. Weme prompts the user to specify at least one field to be edited.
+.. Other incorrect add commands to try: `edit`, +
+   Expected: Similar to previous.
```

Comment:
The behaviour should be weme prompting invalid command format, which is not the same as the previous one